packages/components/src/color-palette/index.tsx

```diff
@@ -284,22 +342,45 @@ export default function ColorPalette( {
 					) }
 				/>
 			) }
-			<Component
-				clearable={ clearable }
-				clearColor={ clearColor }
-				colors={ colors }
-				onChange={ onChange }
-				value={ value }
-				actions={
-					!! clearable && (
-						<CircularOptionPicker.ButtonAction
-							onClick={ clearColor }
-						>
-							{ __( 'Clear' ) }
-						</CircularOptionPicker.ButtonAction>
-					)
-				}
-			/>
+			{ __experimentalHasMultipleOrigins ? (
+				<MultiplePalettes
+					{ ...paletteCommonProps }
+					colors={ colors as PaletteObject[] }
+				/>
+			) : (
+				<SinglePalette
+					{ ...paletteCommonProps }
+					colors={ colors as ColorObject[] }
+				/>
+			) }
 		</VStack>
 	);
 }
+
+/**
+ * Allows selecting from given colors.
+ *
+ * ```jsx
+ * import { ColorPalette } from '@wordpress/components';
```

Comment:
Copy-pasted verbatim from [README.md](https://github.com/WordPress/gutenberg/blob/5a82cf5723b0feb0de9ec23e267f6b190d2a5ad0/packages/components/src/color-palette/README.md#usage).

Maybe that's not what you were thinking, or maybe we could use more examples.

Reply:
Same as above — it's definitely good enough for now, we can always polish it later

Reply:
Thanks!